File: 7-kyu/weird-words-/python/solution.py
def next_letter(s):
    ret = ''
    for l in s:
        if l.isalpha():
            l_ord = ord(l) + 1
            if l_ord == 91 or l_ord == 123:
                l_ord -= 26
            ret += chr(l_ord)
        else:
            ret += l
    return ret

File: 7-kyu/weird-words-/c++/solution.cpp
#include <cctype>

std::string nextLetter(std::string str)
{
    for (char &c : str) {
        if (std::isalpha(c)) {
            c++;
            if (c == 91 || c == 123) c -= 26;
            // 91 is char after 'Z', 123 is char after 'z'
        }
    }
    return str;
}